net.davidotek.pupgui2.desktop

```diff
@@ -0,0 +1,7 @@
+[Desktop Entry]
```

Comment:
was this submitted upstream?

Reply:
Upstream doesn't use a desktop file as it is generated automatically by AppImageBuilder. Could include it in upstream though.

Reply:
it should be generated/installed by the standard build process. it is specific to neither. a "destkop" app without a `.desktop` file is a bug.

Reply:
Will include it in upstream